packages/frontend/src/components/ControlledAutocomplete/index.tsx

```diff
@@ -143,7 +146,16 @@ function ControlledAutocomplete(
                   colorScheme="secondary"
                   isClearable={!required}
                   items={formComboboxOptions(sessionOptions, showOptionValue)}
-                  onChange={(selectedOption) => onChange(selectedOption)}
+                  onChange={(selectedOption) => {
+                    if (
+                      selectedOption === '' ||
+                      isNaN(Number(selectedOption))
+                    ) {
+                      onChange(selectedOption) // don't parse string or empty option
```

Comment:
nit: captain obvious

Reply:
okok i just felt like in case the if check wasn't clear enough